### PR TITLE
test(suite-native): Import an already imported XPUB

### PR DESCRIPTION
**Pull request by bosomt**
Automatized tests of sanity test Import an already imported XPUB

https://www.notion.so/satoshilabs/Import-an-already-imported-XPUB-c15497dd8e3340618c17029745f11aab?pvs=4

### CODE REVIEW

File: suite-native/app/e2e/pageObjects/accountImportActions.ts [hunk 1]
```diff
@@ -23,6 +23,19 @@ class AccountImportActions {
         await scrollUntilVisible(by.text(accountName));
     }
 
+    async importSameAccount({
```
Comment (ondracja) [inline suite-native/app/e2e/pageObjects/accountImportActions.ts:26]: this method is unnecessary, pls use `importAccount` for this test's purposes.

File: suite-native/app/e2e/pageObjects/accountImportActions.ts [hunk 2]
```diff
@@ -23,6 +23,19 @@ class AccountImportActions {
...
+        networkSymbol,
+        xpub,
+    }: {
+        networkSymbol: string;
+        xpub: string;
+        accountName: string;
+    }) {
+        await this.selectCoin({ networkSymbol });
+        await this.submitXpub({ xpub, isValid: true });
+        await detoxExpect(element(by.text('Coin already synced'))).toBeVisible();
```
Comment (ondracja) [inline suite-native/app/e2e/pageObjects/accountImportActions.ts:36]: While onging expects are fine inside a page action method (eg if you want to assert/wait for something during the test), this is the final assert of the test which should `always` be in the `IT` part of the test, not iside the logic

**Review by ondracja — CHANGES_REQUESTED** [review]

**Review by Nodonisko — CHANGES_REQUESTED** [review]
I think we have this scenarion already covered in `suite-native/app/e2e/tests/invalidAccountsImport.test.ts`

**Comment by bosomt** [comment]
@Nodonisko you are right, i have not noticed it.
@ondracja closing as duplicity

**Pull request closed without merging**